benchmark/BDN.benchmark/Cluster/RespClusterBench.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Embedded.perftest;
+using Garnet.common;
+using Garnet.server;
+
+namespace BDN.benchmark.Cluster
+{
+    public unsafe struct Request
+    {
+        public byte[] buffer;
+        public byte* ptr;
+
+        public Request(int size)
+        {
+            buffer = GC.AllocateArray<byte>(size, pinned: true);
+            ptr = (byte*)Unsafe.AsPointer(ref buffer[0]);
+        }
+    }
+
+    public unsafe class ClusterContext
+    {
+        EmbeddedRespServer server;
+        RespServerSession session;
+        BenchUtils benchUtils = new();
+
+        private readonly int Port = 7000;
+
+        public ReadOnlySpan<byte> keyTag => "{0}"u8;
+        public Request[] singleGetSet;
```

Comment:
Each operation should be 128 GET or SET calls to amortize the overheads across calls to Consume which can be non-trivial at nanosecond scale. See RespParseStress.InlinePine or InlineGet for how we do it elsewhere. Having timings in microsecs instead of nanosecs also makes it easier to reason about and stabilize.

Reply:
Good suggestion, I have made the necessary changes!